migrations/sql/V2021.05.20.13.44__update_now_submissions_mechanical_trenching.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE now_submissions.mech_trenching_activity ADD COLUMN length numeric;
```

Comment:
I personally prefer a non-conflict column adding
ALTER TABLE <table name> ADD column IF NOT EXISTS <column name >....

Reply:
Updated!